README.md

```diff
@@ -1,10 +1,132 @@
-### sneaker-portfolio
-- `docker-compose up`
+# Sneaker Portfolio  
+<img src = readme-media/gifs/portfolio.gif > </img>
 
-### sneaker-server
-- `poetry run uvicorn api.main:app` to run Backend
 
-### sneaker-fronted
-- `npm install`
-- `npm run build` for building static website for FastAPI
-- `npm start`
+  
+
+# [Check out the Website here!](https://nsli.me)
+
+
+- [Sneaker Portfolio](#sneaker-portfolio)
```

Comment:
What does this mean? Why is it in the index?

Reply:
This was due to auto update of ToC(Table of Contents) in VS code.
 Fixed and added ToC header.